SU-SWS/stanford_profile_helper
# D8CORE-2420: Rebuild router on taxonomy change to fix bug with taxonomy_menu

# READY FOR REVIEW

# Summary
- Rebuilds router on taxonomy menu update in order to fix issue with taxonomy_menu

# Needed By (Date)
- Sept 3rd

# Urgency
- Medium high. It is a pretty bad bug.

# Steps to Test

1. On a fresh build of cardinalsites edit the person types move a nested menu item to the top level (/admin/structure/taxonomy/manage/stanford_person_types/overview)
2. Review /people to see nested item now in filter menu
3. Move item back under another so that it is now nested
4. Review /people and see item still in filter menu 
5. Check out this branch
6. Repeat steps 1 - 4 but now the item should no longer be in the filter menu.

# Affected Projects or Products
- D8CORE
- Lots of sites and reports

# Associated Issues and/or People
- D8CORE-2420

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
